examples/sorted-set-example.php

```diff
@@ -94,6 +94,18 @@ function printBanner(string $message, LoggerInterface $logger): void
     exit(1);
 }
 
+// fetch sorted set by score (0 to 2) in descending order
+$logger->info("Fetching sorted set $SET_NAME by score in descending order\n");
+$response = $client->sortedSetFetchByScore($CACHE_NAME, $SET_NAME, $minScore = 0, $maxScore = 2, $inclusiveMin = true, $inclusiveMax = true, $order=SORT_DESC);
```

Comment:
If you're using named arguments, can't you leave off $inclusiveMin and $inclusiveMax?

Reply:
I thought we want to show named arguments for those too?

Reply:
I was thinking that we would use named arguments to show that you could change sort order without having to specify all of the other arguments, assuming you were using the defaults.